.github/workflows/create_release_pr.yml

```diff
@@ -0,0 +1,33 @@
+name: Create Release PR
+on:
+  workflow_dispatch
+
+jobs:
+  CreateReleasePR:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Determine Release Version
+        id: release_version
+        run: |
+          OUTPUT=$(jq -r '."mongodb-kubernetes-operator"' < $GITHUB_WORKSPACE/release.json)
+          echo "::set-output name=OUTPUT::$OUTPUT"
+          DATE=$(date '+%Y-%m-%d-%H-%M-%S')
+          echo "::set-output name=DATE::$DATE"
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.6'
+          architecture: 'x64'
+      - run: pip install -r requirements.txt
+      - name: Create Release PR
+        uses: technote-space/create-pr-action@v2
+        with:
+          EXECUTE_COMMANDS: |
+            python scripts/ci/update_release.py
```

Comment:
the changes that you do in this file will not be staged at this point, will they be incorporated into the patch?

Reply:
The github action used here will stage the changes after the commands are executed. The only input we need to provide is the commands that make changes.